src/igvfd/schemas/changelogs/model.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`model.json`*
 
+### Minor changes since schema version 3
+
+* Make required properties `lab` and `award` not submittable. No further submission of models are accepted, please submit model sets instead.
```

Comment:
please make the changelog more terse.
lab and award are still submittable, although by admins only.
I would go with: 

* Restrict `lab` and `award` to admins only.

Reply:
Added.